docs/reference/query-dsl/script-score-query.asciidoc

```diff
@@ -162,9 +170,12 @@ between a given query vector and document vectors.
         "match_all": {}
       },
       "script": {
-        "source": "dotProductSparse(params.queryVector, doc['my_sparse_vector'])",
-        "params": {
-          "queryVector": {"2": 0.5, "10" : 111.3, "50": -1.3, "113": 14.8, "4545": 156.0}
+        "source": """
+          double value = dotProductSparse(params.query_vector, doc['my_sparse_vector']);
+          return sigmoid(value, 1, 1);
+        """,
```

Comment:
@jtibshirani  I think this `sigmoid` can also produce negative scores (for example when value = -0.5)?

Reply:
Oops, thanks for catching this! I was trying to implement the the classic [sigmoid function](http://mathworld.wolfram.com/SigmoidFunction.html) `1/(1 + e^(-x))` and misunderstood our function.